src/Common/StackTrace.h

```diff
@@ -38,19 +50,19 @@ class StackTrace
 
     size_t getSize() const;
     size_t getOffset() const;
-    const Frames & getFrames() const;
+    const FramePointers & getFramePointers() const;
     std::string toString() const;
 
-    static std::string toString(void ** frames, size_t offset, size_t size);
+    static std::string toString(void ** frame_pointers, size_t offset, size_t size);
+    static void symbolize(const void * const * frame_pointers, size_t offset, size_t size, StackTrace::Frames & frames);
```

Comment:
The type of frame_pointers argument is different than in toString function but it does not make sense and can confuse reader.

Reply:
> The type of frame_pointers argument is different than in toString function but it does not make sense and can confuse reader.

@alexey-milovidov It didn't compile without second const on pointer returned from `std::array`'s `data()` method.